test_scripts/Defects/5_0/2670_4_GetInteriorVD_after_ignition_off.lua

```diff
@@ -43,8 +42,8 @@ local function updatePreloadedPT()
   utils.tableToJsonFile(preloadedTable, preloadedFile)
 end
 
-function preconditions()
```

Comment:
Seems that this test corrupts the preloaded policy table, it is not restored after completion

Reply:
@jacobkeeler Good catch. 
We didn't find this since we use external script runner on CI which performs additional backup/restore of SDL source files. 
Please find update in a520d88.